### PR TITLE
Fixed DynamoDB tests

### PR DESCRIPTION
**Pull request by Nyholm**
I dont know how this have worked before...

### CODE REVIEW

File: src/Service/DynamoDb/tests/Integration/DynamoDbClientTest.php [hunk 1]
```diff
@@ -380,7 +381,7 @@ public function testTableNotExists(): void
         $client = $this->getClient();
 
         $input = new DescribeTableInput([
-            'TableName' => $this->tableName,
+            'TableName' => $this->tableName.'_non_exist',
```
Comment (jderusse) [inline src/Service/DynamoDb/tests/Integration/DynamoDbClientTest.php:384]: This is wrong, the test asserts that `tableNotExists($input)` is false. which mean the table should exists

File: src/Service/DynamoDb/tests/Integration/DynamoDbClientTest.php [hunk 2]
```diff
@@ -414,6 +415,6 @@ private function getClient(): DynamoDbClient
 
         return $this->client = new DynamoDbClient([
             'endpoint' => 'http://localhost:8000',
-        ], new Credentials('aws_id', 'aws_secret'));
+        ], new Credentials('aws_id', 'aws_secret'), $httpClient = HttpClient::create());
```
Comment (jderusse) [inline src/Service/DynamoDb/tests/Integration/DynamoDbClientTest.php:418]: I suggest to keep the default client to see if there is something wrong with the retryiableHttpClient

File: src/Service/DynamoDb/tests/Integration/DynamoDbClientTest.php (hunk 1 above)
Comment (Nyholm) [reply src/Service/DynamoDb/tests/Integration/DynamoDbClientTest.php]: You are correct. I didnt see that until now. 

Thank you

File: src/Service/DynamoDb/tests/Integration/DynamoDbClientTest.php (hunk 2 above)
Comment (Nyholm) [reply src/Service/DynamoDb/tests/Integration/DynamoDbClientTest.php:418]: I agree. This is just me debugging. I failed to start the docker locally....

**Pull request closed without merging**